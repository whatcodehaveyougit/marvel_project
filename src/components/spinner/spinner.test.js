import { render, fireEvent, screen } from '@testing-library/react';
import Spinner from './spinner.component';

describe('Spinner', () => {

  it('renders correctly', () => {
    const tree = render(<Spinner />).asFragment();
    expect(tree).toMatchSnapshot();
  });

})