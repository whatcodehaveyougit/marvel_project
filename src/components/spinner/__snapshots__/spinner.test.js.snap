// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner renders correctly 1`] = `
<DocumentFragment>
  <div
    class="fixed inset-0 flex items-center justify-center bg-white bg-opacity-90"
    datatest-id="spinner"
  >
    <div
      class="w-16 h-16 border-8 border-blue-500 border-t-transparent border-r-transparent rounded-full animate-spin"
    />
  </div>
</DocumentFragment>
`;
